ic-os/components/selinux/systemd-fixes/systemd-fixes.te

```diff
@@ -10,6 +10,10 @@ policy_module(systemd-fixes, 1.0.0)
 require { type init_t, initrc_t; }
 allow init_t initrc_t : process2 { nnp_transition };
 
+# Allow the NoNewPrivileges transition for fstrim_tool service that will call fstrim
+require { type fsadm_t; }
+allow initrc_t fsadm_t : process2 { nnp_transition };
+
```

Comment:
Is this just to fix denial logs, or does the [node decommissioning stage 2](https://github.com/dfinity/ic/pull/2953) solution not work without this change?

Reply:
I just saw this comment https://dfinity.atlassian.net/browse/NODE-1522?focusedCommentId=172980 😃

Reply:
This is only warning, if a `systemd` service fails to restrict its privileges further, it still operates as expected, which was confirmed manually on a Farm testnet.